source/_drafts/Introducing-Video-js-HTTP-Streaming-VHS.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Introducing Video.js HTTP Streaming (VHS)
+author:
+  name: Garrett Singer
+  github: gesinger
+tags:
+  - vhs
+  - videojs-http-streaming
+  - http-streaming
+  - playback
+---
+
+> "How do I get my video to play with Video.js?"
+
+This is one of the most frequent questions we get when working on Video.js. And it's a good question.
+
+If someone checks out a copy of Video.js, their content may play on one browser and not another. To get cross browser support for a specific type of content, they must stumble upon a relevant source handler (for instance, [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) for HLS content). Only after including the source handler will their video play across all major browsers.
+
+We want Video.js to be easy to use, and that isn't the easiest setup. So we decided that it's time to address the issue.
+
+This was the motivation behind integrating [Video.js HTTP Streaming](https://github.com/videojs/http-streaming) (nicknamed VHS) inside of [Video.js 7 by default](https://blog.videojs.com/video-js-7-is-here/).
+
+## What is VHS?
+
+![](./vhs-logo.svg)
+
+VHS is the successor to videojs-contrib-hls. It is a source handler forked from the [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) repository. While videojs-contrib-hls was originally designed to add HLS playback on all browsers, we realized that the engine could also play other formats just as well.
+
+To prove this, we added a [DASH manifest parser](https://github.com/videojs/mpd-parser), and with a few minor changes, VHS played DASH content with the same codebase and API as it used for HLS.
+
+With the changing landscape of video technologies, if a new format comes around and gains popularity, with the addition of a small manifest parser and maybe a few minor code changes, we expect it to play via VHS as well. One engine and API for all formats.
```

Comment:
I would still rephrase the latter part of this sentence "with the addition of..." onwards

Reply:
Yeah, the second part is a big awkward.

Reply:
Updated. Thoughts?